### PR TITLE
[12.0][ADD]purchase_order_incoterm_extend

### PR DESCRIPTION
**Pull request by TomohiroAbe-Quartile**
Add the "Incoterm Place" field to purchase order as supplementary information of the selected incoterm. The input value will show in the printed purchase order / incoterms.
And enable Incoterm output on Purchase Order app.

### CODE REVIEW

**Comment by yostashiro** [comment]
@TomohiroAbe-Quartile Can you also take care of the purchase.report_purchasequotation template?

File: purchase_order_incoterm_extend/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This module does the following:
+
+- Add the "Incoterm Place" field to purchase order as supplementary information of the selected incoterm. The input value will show in the printed purchase order / incoterms.
```
Comment (yostashiro) [inline purchase_order_incoterm_extend/readme/DESCRIPTION.rst]:
```suggestion
- Adds the "Incoterm Place" field to purchase order as supplementary information of the selected incoterm.
  The input value will show in the printed purchase order, together with the incoterm code.
```

File: purchase_order_incoterm_extend/report/purchase_order_report_templates.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document_inherit_purchase_order"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//div[@t-if='o.date_order']" position="after">
+            <div t-if="o.incoterm_id" class="col-3 bm-2">
```
Comment (yostashiro) [inline purchase_order_incoterm_extend/report/purchase_order_report_templates.xml]: Can't find the `bm-2` definition anywhere in the code. Maybe we should remove this?

File: purchase_order_incoterm_extend/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```
Comment (yostashiro) [inline purchase_order_incoterm_extend/tests/__init__.py]: Remove this line.

File: purchase_order_incoterm_extend/models/purchase_order.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Quartile Limited
```
Comment (yostashiro) [inline purchase_order_incoterm_extend/models/purchase_order.py]:
```suggestion
# Copyright 2021 Quartile Limited
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
I don't think tests are needed since there is no logic involved.

File: purchase_order_incoterm_extend/tests/__init__.py (hunk 3 above)
Comment (TomohiroAbe-Quartile) [reply purchase_order_incoterm_extend/tests/__init__.py]: Removed. Thanks.

File: purchase_order_incoterm_extend/models/purchase_order.py (hunk 4 above)
Comment (TomohiroAbe-Quartile) [reply purchase_order_incoterm_extend/models/purchase_order.py]: Edit it. Thank you.

File: purchase_order_incoterm_extend/report/purchase_order_report_templates.xml (hunk 2 above)
Comment (TomohiroAbe-Quartile) [reply purchase_order_incoterm_extend/report/purchase_order_report_templates.xml]: It removed but works. Thank you.

File: purchase_order_incoterm_extend/readme/DESCRIPTION.rst (hunk 1 above)
Comment (TomohiroAbe-Quartile) [reply purchase_order_incoterm_extend/readme/DESCRIPTION.rst]: Edit it. Thank you.

**Comment by TomohiroAbe-Quartile** [comment]
@yostashiro I delete test code. Thank you.

**Pull request closed without merging**